packages/appstore/src/components/containers/listing-container.tsx

```diff
@@ -23,9 +29,10 @@ const ListingContainer = ({
                 </div>
                 {is_deriv_platform && <SelectedAccountCard />}
             </div>
+            {!is_deriv_platform && !is_demo && !has_any_real_account && <AddOptionsAccount />}
```

Comment:
I suggest using `truthy` conditions for better code readability.
example check for `is_real` instead of `!is_demo`.
same for `!is_deriv_platform `